extensions/amp-analytics/0.1/vendors/snowplow_v2.json

```diff
@@ -10,23 +10,24 @@
   "cookies": {
     "enabled": true,
     "_sp_duid": {
-      "value": "$IF($IF(COOKIE(_sp_duid), COOKIE(_sp_duid), LINKER_PARAM(linker,_sp_duid)), $IF(COOKIE(_sp_duid), COOKIE(_sp_duid), LINKER_PARAM(linker,_sp_duid)), $SUBSTR(QUERY_PARAM(_sp),0,36))"
+      "value": "$IF($SUBSTR(QUERY_PARAM(_sp),0,36), $SUBSTR(QUERY_PARAM(_sp),0,36), $IF(LINKER_PARAM(sp_amp_linker,_sp_duid), LINKER_PARAM(sp_amp_linker,_sp_duid), COOKIE(_sp_duid)))"
```

Comment:
Can you use the duid var here instead of duplicating this logic?

Reply:
Yes we can. That'd be way simpler. 🤦

Reply:
Scratch that. We can't, the cookie part doesn't sub out the var. Which must be why it was like this in the first place.